sphinx_rtd_theme/theme.conf

```diff
@@ -12,3 +12,4 @@ display_version = True
 navigation_depth = 4
 prev_next_buttons_location = bottom
 canonical_url =
+titles_only = True
```

Comment:
I guess the default value should likely be kept to `False` as it used to be?

Reply:
Ok, I looked into this and that option is being read in as a string. So, it doesn't matter if you put "True" or "False", it'll still read True since they're both strings. An empty string will register as False though. 

See here: https://github.com/sphinx-doc/sphinx/blob/cb0220de1a0d8acd00292b898d2ef03b588179b0/sphinx/environment/adapters/toctree.py#L180

The True / False setting in conf.py works because those are actually read in as booleans.